docs/host/host.md

```diff
@@ -23,6 +23,17 @@ Because Harvester is built on top of Kubernetes and uses etcd as its database, t
 
 For admin users, you can click **Enable Maintenance Mode** to evict all VMs from a node automatically. It will leverage the `VM live migration` feature to migrate all VMs to other nodes automatically. Note that at least two active nodes are required to use this feature.
 
+If you want to individually force a shutdown of a VM instead of migrating it to another node, add the label `harvesterhci.io/maintain-mode-strategy` and one of these values to those VMs:
+
+- `Migrate`: The VM will be live-migrated to another node in the cluster. This is the default behavior if the label `harvesterhci.io/maintain-mode-strategy` is not set.
+- `ShutdownAndRestartAfterEnable`: Restart the VM after the node has finally switched into maintenance mode. The VM will be scheduled on a different node.
+- `ShutdownAndRestartAfterDisable`: Shut down when maintenance mode is enabled and restart the VM after maintenance mode is disabled. The VM will stay on the same node.
+- `Shutdown`: Shut down when maintenance mode is enabled. Do NOT restart the VM, it remains switched off.
```

Comment:
```suggestion
If you want to force individual VMs to shut down instead of migrating to other nodes, add the label `harvesterhci.io/maintain-mode-strategy` and one of the following values to those VMs:

- `Migrate`: Live-migrates the VM to another node in the cluster. This is the default behavior if the label `harvesterhci.io/maintain-mode-strategy` is not set.
- `ShutdownAndRestartAfterEnable`: Restarts the VM after the node switches to maintenance mode. The VM is scheduled on a different node.
- `ShutdownAndRestartAfterDisable`: Shuts down the VM when maintenance mode is enabled, and restarts the VM when maintenance mode is disabled. The VM stays on the same node.
- `Shutdown`: Shuts down the VM when maintenance mode is enabled. The VM must remain powered off so do **not** restart it.
```

Reply:
# Old
- `Shutdown`: Shut down when maintenance mode is enabled. Do NOT restart the VM, it remains switched off.

# Siggested
- `Shutdown`: Shuts down the VM when maintenance mode is enabled. The VM must remain powered off so do **not** restart it.

what i meant here should be understood as a hint that the VM remains switched off and not as an instruction that the user may not restart the VM.

Reply:
Got it. Here is the reworded description: 

`Shutdown`: Shuts down the VM when maintenance mode is enabled. The VM remains powered off instead of restarting.